fiasko_bro/code_validator.py

```diff
@@ -9,6 +10,24 @@ def validate_repo(path_to_repo, path_to_original_repo=None):
     return code_validator.validate(path_to_repo, path_to_original_repo)
 
 
+def py_files_count(path_to_repo):
+    counter = 0
+    for _, _, files in os.walk(path_to_repo):
+        for file in files:
```

Comment:
Don't shadow built-in names. In fact, these are not `files`, but mere `filenames`.